cli/src/semgrep/config_resolver.py

```diff
@@ -249,7 +251,14 @@ def parse_config_files(
     but is None for registry rules
     """
     config = {}
-    for config_id, contents, config_path in loaded_config_infos:
+    for config_id, contents, config_path in progress.track(
+        loaded_config_infos,
+        description=f"  parsing {len(loaded_config_infos)} rules",
+        transient=True,
+        # expected to take just 2-3 seconds with less than 500
```

Comment:
I wouldn't mind a three second progress bar. Or even a 0.5 second one. I don't feel strongly enough to request a change but I'm not sure there's any point in disabling it.

Reply:
Yeah that's reasonable